src/java.base/share/classes/java/time/chrono/IsoChronology.java

```diff
@@ -677,6 +677,19 @@ public Period period(int years, int months, int days) {
         return Period.of(years, months, days);
     }
 
+    //-----------------------------------------------------------------------
+    /**
+     * {@code IsoChronology} is an ISO based chronology, which supports fields,
+     * such as {@code DAY_OF_QUARTER} and {@code QUARTER_OF_YEAR}.
+     * @implSpec The implementation always returns {@code true}.
+     * @return {@code true}
+     * @since 19
+     */
+    @Override
+    public boolean isIsoBased() {
```

Comment:
Is this meant to be 'default'?  The CSR indicated adding default methods.

Reply:
The `default` method has been added to `java.time.chrono.Chronology` interface. This is its overridden method implemented in `IsoChronology` concrete class.